contracts/lifecycle/DestructibleDelayed.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^ 0.4.24;
+
+import "../ownership/Ownable.sol";
+
+/**
+ * @title DestructibleDelayed
+ * @dev Contract that can be selfdestructed after a delay period.
+ */
+
+
+contract DestructibleDelayed is Ownable {
+
+  // Whether a request for the self-destruction of the contract was requested.
+  bool public destructionRequested = false;
+
+  // When self-destruction is possible (UNIX timestamp)
+  uint256 public destructionTime = 2**256-1; 
+
+  // Minimum delay after which destroy() can be called once requested
+  uint256 private SELFDESTRUCTION_DELAY; 
+
+  // Events
+  event SelfDestructionRequest(uint256 _destructionTime);
```

Comment:
Please phrase this as a verb in past tense (i.e. `SelfDestructionRequested`). That'll be described in a upcoming naming style document :)